docs/source/tutorials_source/platform/tutorial_cropped_objects_metadata.py

```diff
@@ -0,0 +1,166 @@
+
+    BASE_PATH = "path/to/dataset/"
+    DATASET_PATH = BASE_PATH+"images"  # the path were the full images are found
+    OUTPUT_DIR = BASE_PATH+"cropped_images" # the path where the cropped images will be saved
+    # the file where the objectness scores will be saved
+    METADATA_OUTPUT_FILE = BASE_PATH+"cropped_images_objectness_scores.json"
+
+    ''' 1. Define the dataset and dataloader'''
+    x_size = 2048
+    y_size = 2048
+    transform = torchvision.transforms.Compose([
+        torchvision.transforms.Resize((x_size, y_size)),
+            torchvision.transforms.ToTensor(),
+            torchvision.transforms.Normalize(
+                mean=[0.485, 0.456, 0.406],
+                std=[0.229, 0.224, 0.225])
+        ])
+    dataset = LightlyDataset(DATASET_PATH, transform=transform)
+
+    ''' 2. Define the pretrained object detection model'''
+    model = torchvision.models.detection.retinanet_resnet50_fpn(pretrained=True)
+
+    ''' 3. Predict with the model on the dataset '''
+    model.eval()
+    dataloader = DataLoader(dataset, batch_size=2)
+    predictions = []
+    with torch.no_grad():
+        for x, _, _ in tqdm(dataloader):
+            pred = model(x)
+            predictions.append(pred)
+
+    predictions = [i for sublist in predictions for i in sublist]
+
+    ''' 4. Save the cropped objects '''
+    class_indices_list_list = [list(prediction["labels"]) for prediction in predictions]
+    bounding_boxes_list_list = []
+    for prediction in predictions:
+        bounding_boxes_list = []
+        for box in prediction["boxes"]:
+            x0 = box[0] / x_size
+            y0 = box[1] / y_size
+            x1 = box[2] / x_size
+            y1 = box[3] / y_size
+            bounding_boxes_list.append(BoundingBox(x0, y0, x1, y1))
+        bounding_boxes_list_list.append(bounding_boxes_list)
+
+    cropped_images_list_list = crop_dataset_by_bounding_boxes_and_save(dataset, OUTPUT_DIR,
+        bounding_boxes_list_list, class_indices_list_list)
+
+    '''  5. Save the objectness scores as metadata '''
+    objectness_scores_list_list = [list(prediction["scores"]) for prediction in predictions]
+    metadata_list = []
+    for cropped_images_list, objectness_scores_list in zip(cropped_images_list_list, objectness_scores_list_list):
+        for cropped_images_filename, objectness_score in zip(cropped_images_list, objectness_scores_list):
+            metadata = {"objectness_score": float(objectness_score)}
+            metadata_list.append((cropped_images_filename, metadata))
+    save_custom_metadata(METADATA_OUTPUT_FILE, metadata_list)
+
+    ''' 6. Tell the lightly CLI command '''
+    cli_command = f"lightly-magic input_dir={OUTPUT_DIR} new_dataset_name=SKU_110k_val_cropped trainer.max_epochs=0 "
+    cli_command += f"custom_metadata={METADATA_OUTPUT_FILE} token=MY_TOKEN"
+    print(f"Upload the images and custom metadata with the following CLI command:")
+    print(f"{bcolors.OKBLUE}{cli_command}{bcolors.ENDC}")
+
```

Comment:
Can we add a screenshot here on how the output of the terminal could look like? You can black out the token and dataset id using gimp

Reply:
I added one, but it does not look to good in my eyes.
If you don't disagree I would remove the screenshot again. It is only the usual output of a `lightly-magic` command, it would fit better to a general `lightly-magic` tutorial or documentation in my eyes.

Reply:
I added a screenshot, but it does not look too good. I would remove it, I don't think it provides a lot of value, as it is just the usual lightly-magic output.

Reply:
I think the screenshot adds value but the text is too small. Any chance in doing the same with a smaller terminal window or using a zoomed terminal window?
Don't forget that this also acts as a check whether all the steps before were done correctly. If I run the tutorial and get a similar output I know that I'm on track.

Reply:
I added a screenshot with a bigger text.